libfm-qt/pathedit.cpp

```diff
@@ -111,9 +111,11 @@ void PathEdit::reloadCompleter(bool triggeredByFocusInEvent) {
   // qDebug("load: %s", g_file_get_uri(data->dirName));
   cancellable_ = g_cancellable_new();
   data->cancellable = (GCancellable*)g_object_ref(cancellable_);
-  g_io_scheduler_push_job((GIOSchedulerJobFunc)jobFunc,
-                          data, (GDestroyNotify)JobData::freeMe,
-                          G_PRIORITY_LOW, cancellable_);
+
+  GTask *task = g_task_new(NULL, NULL, NULL, NULL);
```

Comment:
Alright then, what do you propose ? Keep the old code between `#if` (which version ?) `#endif`  to maintain backward compatibility or let the new code in standby as long as the deprecated function is present ?